Resources/public/javascript/sticky.js

```diff
@@ -0,0 +1,34 @@
+require('@wearejust/sticky');
+
+let addBtnClone;
+let elOffsetLeft;
+let $window = $(window);
+let minWidth = 940;
+
+$(function() {
+	if($window.width() > minWidth) {
+		$('.add_records').attr('data-sticky-id', 'sticky_add_btn');
+		elOffsetLeft = $('.add_records').offset().left;
+		addBtnClone = $('.add_records').clone().appendTo('.navbar-collapse');
```

Comment:
Liever de clone al in de HTML zetten en hier selecteren, dan dat je hem hier clonet en helemaal aanpast (classes op/af, etc).